backend/utilities/auth.py

```
@@ -85,31 +89,38 @@ async def get_current_user(
+        logger.warning("Authentication failed: No user ID in token payload")
         raise credentials_exception
     
     try:
         user_id = int(user_id_str)
     except (ValueError, TypeError):
+        logger.warning("Authentication failed: Invalid user ID format")
         raise credentials_exception
     
     user = db.query(User).filter(User.id == user_id).first()
     if user is None:
+        logger.warning(f"Authentication failed: User not found with ID: {user_id}")
```

Ruff (0.14.1)

Logging statement uses f-string

(G004)